app/src/main/java/com/framgia/fsalon/screen/customer/CustomerPresenter.java

```diff
@@ -1,15 +1,40 @@
 package com.framgia.fsalon.screen.customer;
 
+import android.util.Log;
+
+import com.framgia.fsalon.data.model.BookingCustomer;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.data.model.BookingCustomer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports